etc/config/hook.amazon.properties

```diff
@@ -0,0 +1,20 @@
+
+interpreted=true
+supportsBinary=false
+supportsExecute=true
+versionFlag=--version
+needsMulti=false
+
+compilerType=hook
+compilers=&hook
+defaultCompiler=hook010
+
+group.hook.compilers=hook010
+group.hook.isSemVer=true
+group.hook.baseName=Hook
+group.hook.licenseName=MIT
+group.hook.licenseLink=https://github.com/fabiosvm/hook-lang/blob/main/LICENSE
+
+compiler.hook010.exe=/opt/compiler-explorer/hook-0.1.0/bin/hook
+compiler.hook010.semver=0.1.0
+compiler.hook010.name=Hook 0.1.0
```

Comment:
Sorry that I keep finding different issues with the config each time instead of all at once :|

You don't need to define a `.name` key for `isSemVer=true` compiles, as it's automatically set to `<baseName> <semver>`.
Also, the group is missing a `groupName` so that its name shows in the header of its section in the compiler dropdown

Reply:
Ok. No problem. 
I removed `.name` and put `groupName`.

About `groupName`, I noticed that `HOOK` already appears in the section header and that there is `.properties` file without `groupName`, eg Python.

But, it's done. Next. ;)